### ISSUE TITLE
Requiring the lib

### ISSUE THREAD

**Issue by ksavenkov** [CassetteRocks/react-infinite-scroller#7]
Hey, I've installed the package via NPM and require it as follows:

`var InfiniteScroll = require('react-infinite-scroller');`

Apparently, something goes wrong as I have a react error:

_React.createElement: type should not be null, undefined, boolean, or number. It should be a string (for DOM elements) or a ReactClass (for composite components). Check the render method of `Search`._

To what I see, the require returns a function. Probably I require it in a wrong way (although I do it in the same way I require other react libs). Maybe the solution is in your phrase "require with main in package.json", but I can't figure out what does it mean. Can you help? Thanks.

**Comment by danbovey** [comment]
Hi

Haven't tested using the library with `require` but it should work. Basically the quick documentation I wrote up about the package.json in simpler terms is, ES6 imports use the file in `jsnext:main` and regular `require`'s like you're using look for the file listed in `main`.

Can you confirm that it's the InfiniteScroll component that is not returning a proper `ReactClass` and not anything else in `Search`.

I'll test this myself later, and going forward I need to implement formal testing that checks both the file in `dist` and the ES6 file in `src`

**Comment by mtolly** [comment]
For whatever reason the actual React class is in the `default` property of the return value of `require`. So this works:

```js
var InfiniteScroll = require('react-infinite-scroller').default;
```

**Comment by danbovey** [comment]
Ah thanks! I clearly need to change the build script that compiles the file in `dist` because this line doesn't look right: [#127](https://github.com/CassetteRocks/react-infinite-scroller/blob/master/dist/react-infinite-scroll.js#L127)
